### DISCUSSION TITLE
About parameters store format in image

### DISCUSSION THREAD

**Discussion by ChiaYen-Kan** [vladmandic/sdnext#3595] [General]
new version is change the parameters store format in image(ex. order, lora hash)
it is don't affect SD.Next, but when post the image to Civitai, Civitai is can't parser parameters from image
is it possible has options to switch to old parameters store format ?

**Comment by vladmandic** [comment]
i'm not going to go to a random place in the past to keep civitai parser happy without knowing whats going on.
if this is going to get resolved, i need to understand civitai parser.
so reach out to their support and loop them in.

**Reply by ChiaYen-Kan** [reply]
parser ok
![OK](https://github.com/user-attachments/assets/926fb1d7-84cf-434a-a891-404eb81e9d37)

parser ng
![NG](https://github.com/user-attachments/assets/575aae29-32ce-4da0-bfe8-b75c3ad1fd07)

i also post to civitai

thank you

**Comment by vladmandic** [comment]
civitai needs to fix their parser, it assumes veeeery old rule that `steps` is *always* the first parameter - which its not.
get in touch with civitai support and connect them to me. i can provide parsing code if they want, but i will not downgrade sdnext to fit their parser.
